indigo_app/static/javascript/indigo/views/document_comparisons.js

```diff
@@ -0,0 +1,66 @@
+(function(exports) {
+  "use strict";
+
+  if (!exports.Indigo) exports.Indigo = {};
+  Indigo = exports.Indigo;
+
+  /**
+   * Compare different expressions (in the same language) of a work.
+   */
+  Indigo.DocumentComparisonView = Backbone.View.extend({
+    el: '.document-comparison-view',
+    comparisonHeaderTemplate: '#comparison-header-template',
+    events: {
+      'click .dismiss': 'dismiss'
+    },
+
+    initialize: function(options) {
+      this.comparisonHeaderTemplate = Handlebars.compile($(this.comparisonHeaderTemplate).html());
+      this.document = options.document;
+      this.documentContent = options.documentContent;
+      this.expressions = new Indigo.Library(Indigo.Preloads.expressions);
+
+      $('.menu .comparison').on('click', _.bind(this.show, this));
+    },
+
+    comparison: function(comparison_doc_id) {
+      var data = {},
+          self = this;
+      data.document = this.documentContent.document.toJSON();
+      data.document.content = this.documentContent.toXml();
+      data.comparison_doc_id = comparison_doc_id;
+      this.compareTo = this.expressions.get(comparison_doc_id);
+
+      $.ajax({
+        url: '/api/document-comparison',
+        type: "POST",
+        data: JSON.stringify(data),
+        contentType: "application/json; charset=utf-8",
+        dataType: "json"})
+          .then(function(response) {
+            self.$el.find('.akoma-ntoso').html(response.content);
+            self.$el.find('.n_changes').html(response.n_changes);
+          });
+    },
+
+    show: function(e) {
+      var comparison_doc_id = e.currentTarget.id;
+      e.preventDefault();
+      $('.work-view').addClass('d-none');
+      this.$el.removeClass('d-none');
+      this.comparison(comparison_doc_id);
+
+      this.$('.title').html(this.comparisonHeaderTemplate({
+        document: this.document,
+        comparison_doc: this.compareTo
```

Comment:
`this.compareTo` is a Backbone model object, which the template won't like. You want to transform it into JSON instead, so you can use dot-notation in the template.

It'll make your life easier if you choose one variable name for a concept, and re-use it wherever you can. eg. "comparisonDoc" vs "compareTo" vs "comparisonDocument" (which I introduced, I guess :)

```suggestion
        comparison_doc: this.compareTo
        compare_to: this.compareTo.toJSON(),
```